### PR TITLE
Feature/silly stuff

### PR DESCRIPTION
**Pull request by jsullivan5**
Really cool feature.  Pull pull this down and let me know what you think. @cammac60 

image

### CODE REVIEW

**Comment by cammac60** [comment]
@jsullivan5 I knew my gallery was missing something, this really provides some pizzaz and uniqueness!

File: src/components/Gallery/Gallery.js [hunk 1]
```diff
@@ -8,12 +8,19 @@ export class Gallery extends Component {
...
     images: [],
+    error: '',
   }
 
   renderImages = () => {
     const images = [];
     const { photoNumber } = this.state;
 
+    if (photoNumber > 1000) {
+      const error = 'Whoa there!  Not that big. Think like <= 1000.'
```
Comment (cammac60) [inline src/components/Gallery/Gallery.js:19]: I think we should consider allowing over 1000 cats. User testing indicates that 5% of the users scroll through more than 2.5k cats per hour.

File: src/components/Gallery/Gallery.js [hunk 2]
```diff
@@ -1,45 +1,114 @@
...
+  handleSubmit = (event) => {
+    event.preventDefault();
+    this.renderImages();
+  }
+
+  render() {
+    const { photoNumber, images, error } = this.state;
+
+    return (
+      <div className="Gallery">
```
Comment (cammac60) [inline src/components/Gallery/Gallery.js:56]: Needs more wrappers.

**Review by cammac60 — COMMENTED** [review]
Needs more cats.

File: src/components/Gallery/Gallery.js (hunk 1 above)
Comment (jsullivan5) [reply src/components/Gallery/Gallery.js:19]: Believe me I tried.  But the page was having performance issues loading that many cats.  I think when we deploy this to the cloud we should mirror the cats on a highly available CDN or perhaps introduce a Redus caching layer in the Kubernetes cluster.  :)

**Pull request closed without merging**